migrations/20240916110123_create_items_table.js

```diff
@@ -0,0 +1,40 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('items', function (table) {
+    table.increments('id').primary();
+    table
+      .integer('owner_id')
+      .unsigned()
+      .notNullable()
+      .references('id')
+      .inTable('users')
+      .onDelete('CASCADE');
+    table
+      .integer('bucket_id')
+      .unsigned()
+      .notNullable()
+      .references('id')
+      .inTable('buckets')
+      .onDelete('CASCADE');
+    table
+      .integer('parent_id')
+      .unsigned()
+      .nullable()
+      .references('id')
+      .inTable('items')
+      .onDelete('SET NULL');
+    table.string('name', 255).notNullable();
+    table.enu('item_type', ['folder', 'file']).notNullable();
+    table.string('path', 512).nullable();
+    table.string('filetype', 50).nullable();
+    table.bigInteger('size').nullable();
```

Comment:
```suggestion
    table.integer('size').nullable();
```

We don't need to save the file's size as byte. We can convert it to MB.